token/cli/src/main.rs

```diff
@@ -4290,7 +4338,7 @@ mod tests {
 
         let token = create_token(&config, &payer).await;
         let source = create_associated_account(&config, &payer, token).await;
-        let recipient = token.to_string();
+        let recipient = Keypair::new().pubkey().to_string();
```

Comment:
i changed this because the errors i added to `command_transfer` break it. however i think this was done just as a convenience, and there isnt a real usecase for automatically creating an associated account owned by a mint